spec/board_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require './lib/board'
+require './lib/validation'
```

Comment:
- **[OPTIONAL]** Using `require_relative` for files in your directory instead of `require` is a ruby convention.

> "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."